resources/ko-KR/dialogs/nlg-strings.dialog.bxb

```diff
@@ -91,6 +96,85 @@ template-macro-def (SPACE_RESORT_RESULT) {
   }
 }
 
+template-macro-def (SPACE_RESORT_NAME_WITH_EMPHASIS) {
```

Comment:
YT for translation: https://sixfivelabs.myjetbrains.com/youtrack/issue/VIV-25644